src/components/customerPortal/usage/UsagePage.tsx

```diff
@@ -88,6 +89,10 @@ const UsagePage = () => {
   const { translate, documentLocale } = useCustomerPortalTranslate()
   const customerId = 'cdef1dac-c55f-4d25-985b-cb25c2c8edc1'
   const { itemId } = useParams()
+  const [activeTab, setActiveTab] = useState<number>(0)
```

Comment:
can we use an enum instead of a number to make it more readable? 

```suggestion
enum UsageTab {
 // named tab
}

 const [activeTab, setActiveTab] = useState<UsageTab>(UsageTab...)
```

Reply:
We're constrained a bit by the

`<NavigationTab>` component which uses `onChange={(index) => setActiveTab(index)}`